NuCivic/dkan
# Add hook_update to disable default content on upgrade to 1.13 - CIVIC-4890

Issue: civic-4890

## Description

Due to the overhaul on dkan_default_content, existing sites should not have this module enabled after an upgrade since it will try to run imports that require dkan_fixtures and will throw a `class not found` error when trying to enable other modules.

## QA Tests

```
git checkout tags/7.x-1.12.11
ahoy dkan remake && ahoy dkan reinstall
git checkout 7.x-1.x
ahoy dkan remake
ahoy drush rr
ahoy drush en dkan_ipe -y
ahoy drush en dkan_harvest_dashboard -y
ahoy drush en menu_admin_per_menu -y
ahoy drush fra -y
ahoy drush updatedb -y
ahoy drush rr
```
Confirm the dkan_sitewide_default_content feature is not enabled.
